docs/user-guides/workspace-access/remote-desktops.md

```diff
@@ -15,37 +15,41 @@ Installation instructions vary depending on your workspace's operating system,
 platform, and build system.
 
 As a starting point, see the
-[desktop-container](https://github.com/bpmct/coder-templates/tree/main/desktop-container)
-community template. It builds and provisions a Dockerized workspace with the
+[enterprise-desktop](https://github.com/coder/images/tree/main/images/desktop)
+image. It can be used to provision a Dockerized workspace with the
 following software:
 
-- Ubuntu 20.04
-- TigerVNC server
-- noVNC client
+- Ubuntu 24.04
 - XFCE Desktop
+- KasmVNC Server and Web Client
 
 ## RDP Desktop
 
 To use RDP with Coder, you'll need to install an
 [RDP client](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-clients)
 on your local machine, and enable RDP on your workspace.
 
-Use the following command to forward the RDP port to your local machine:
+<div class="tabs">
 
-```console
-coder port-forward <workspace-name> --tcp 3399:3389
-```
+### Coder Desktop
```

Comment:
Needs to be rebased on https://github.com/coder/coder/pull/18354 and consolidated.